weave-js/src/components/PagePanelComponents/Home/Browse3/typeViews/Audio/AudioPlayer.tsx

```diff
@@ -56,7 +56,9 @@ export const AudioPlayer: FC<{
   }
 
   const downloadFile = () => {
-    if (!audioUrl || !audioFileName) return;
+    if (!audioUrl) {
+      return console.error('Audio URL is not available');
```

Comment:
what happens when you return a console error? shouldn't you error, then return None?

Reply:
console.error is a void function so it shouldn't do anything but I'll change it just in case

Reply:
yeah ig, just looks weird